src/Lean/Elab/Extra.lean

```diff
@@ -79,13 +79,13 @@ private def throwForInFailure (forInInstance : Expr) : TermElabM Expr :=
 namespace Op
 /-!
 
-The elaborator for `binop%`, `binop_lazy%`, and `unop%` terms.
+The elaborator for `binop%`, `binop_lazy%`, `leftact%`, `rightact%`, and `unop%` terms.
 
```

Comment:
Could you please add a summary describing each one of the operators?
We need to raise the bar for files being touched by multiple people.

Reply:
Recall we originally only had `binop%`, and the lists keeps getting bigger :)

Reply:
I'm just on my phone and I haven't checked what got included, but I had previously worked out what would be needed to PR this to core and included additional docstings in a few places: https://github.com/leanprover/lean4/compare/master...kmill:kmill_binop2?expand=1

Reply:
@kmill Let's get them in as soon as you start next week.

Reply:
I expanded the docstring and added summaries (Sebastian and I decided that I'd create a new PR since I can't push to this branch, so it's over at #2778)